containers/tefca-viewer/src/app/CustomQuery.ts

```diff
@@ -75,7 +75,7 @@ export class CustomQuery {
         : "";
     this.medicationRequestQuery =
       rxnormFilter !== ""
-        ? `/MedicationRequest?subject=${patientId}&code=${rxnormFilter}&_include=MedicationRequest:medication&_include=MedicationRequest:medication.administration`
+        ? `/MedicationRequest?subject=${patientId}&code=${rxnormFilter}&_include=MedicationRequest:medication&_include=MedicationRequest:intended-performer`
```

Comment:
For some reason, Public HAPI didn't like `medication.administration` as an includes param, which was breaking the inclusion of the medication requests for the patient. I did a bit of sparse googling and _think_ this is the closest of the provided options below from the error message but if someone knows better lmk and I'm happy to change it.

```
Invalid _include parameter value: &quot;MedicationRequest:medication.administration&quot;. Unknown search parameter &quot;medication.administration&quot; for resource type &quot;MedicationRequest&quot;. Valid search parameters for this search are: [encounter, intended-dispenser, intended-performer, medication, patient, requester, subject]
```